src/ImageService.Broker/Consumers/GetImagesConsumer.cs

```diff
@@ -1,128 +1,128 @@
-﻿using LT.DigitalOffice.ImageService.Data.Interfaces;
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LT.DigitalOffice.ImageService.Data.Interfaces;
 using LT.DigitalOffice.ImageService.Models.Db;
 using LT.DigitalOffice.Kernel.Broker;
 using LT.DigitalOffice.Models.Broker.Enums;
 using LT.DigitalOffice.Models.Broker.Models;
 using LT.DigitalOffice.Models.Broker.Requests.Image;
 using LT.DigitalOffice.Models.Broker.Responses.Image;
 using MassTransit;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace LT.DigitalOffice.ImageService.Broker.Consumers
 {
-    public class GetImagesConsumer : IConsumer<IGetImagesRequest>
+  public class GetImagesConsumer : IConsumer<IGetImagesRequest>
+  {
+    private readonly IImageNewsRepository _imageNewsRepository;
```

Comment:
we do not need get news images throw consumer

Reply:
done